frontend/src/components/associations/marketplace/QuantitySelect.tsx

```diff
@@ -2,30 +2,26 @@ import React, { useState } from "react";
 import { Button } from "react-bootstrap";
 
 export const QuantitySelect = ({ order }) => {
-    let [quantity, setQuantity] = useState(0);
+    const [quantity, setQuantity] = useState(0);
 
-    let increase = () => {
+    const increase = () => {
         setQuantity(quantity + 1);
     };
-    let decrease = () => {
+    const decrease = () => {
         setQuantity(quantity - 1);
     };
 
-    let onOrder = () => {
+    const onOrder = () => {
         order(quantity);
         setQuantity(0);
     };
 
-    if (quantity === 0) {
-        return (
-            <Button variant="primary" onClick={increase}>
-                <i className="fe fe-plus" />
-                Commander
-            </Button>
-        );
-    }
-
-    return (
+    return quantity === 0 ? (
```

Comment:
It's big but it doesn't take space that should be attributed to something else so it's not an issue. Also it would be unnecessarily complicated to make an input of the length of 3 or 4 characters